adcc/testdata/generate_psi4_data.py

```diff
@@ -0,0 +1,97 @@
+import psi4
+from psi4.driver.procrouting.response.scf_response import tdscf_excitations
+from adcc.testdata import static_data
+import yaml
+import os
+
+
+def run_psi4_tdscf(xyz, basis, charge=0, multiplicity=1,
+                   conv_tol=1e-12, conv_tol_grad=1e-11, max_iter=150,
+                   pcm_options=None):
+    mol = psi4.geometry(f"""
+        {charge} {multiplicity}
+        {xyz}
+        units au
+        symmetry c1
+    """)
+    psi4.set_options({
+        'basis': basis,
+        'scf_type': "pK",
+        'e_convergence': conv_tol,
+        'd_convergence': conv_tol_grad,
+        'maxiter': max_iter,
+        'reference': "RHF",
+        'save_jk': True,
+    })
+    if pcm_options:
+        # think its better to use a dict for the pcm options, because there
+        # are already enough function arguments. Of course one could just
+        # fix the options... would be sufficient for now too.
+        psi4.set_options({'pcm': True, 'pcm_scf_type': "total"})
+        psi4.pcm_helper(f"""
+            Units = AU
+            Cavity {{
+                Type = Gepol
+                Area = {pcm_options["weight"]}
+            }}
+            Medium {{
+                Solvertype = {pcm_options["pcm_method"]}
+                Solvent = {pcm_options["solvent"]}
+                Nonequilibrium = {pcm_options["neq"]}
+            }}
+        """)
+    psi4.core.be_quiet()
+    _, wfn = psi4.energy('scf', return_wfn=True, molecule=mol)
+
+    res = tdscf_excitations(wfn, states=5, triplets="NONE", tda=True,
+                            r_convergence=1e-7)
+
+    # remove cavity files from PSI4 PCM calculations
+    if pcm_options:
+        for cavityfile in os.listdir(os.getcwd()):
+            if cavityfile.startswith(("cavity.off_", "PEDRA.OUT_")):
+                os.remove(cavityfile)
+    return wfn, res
+
+
+def dump_results(molecule, basis, **kwargs):
+    pcm_options = kwargs.get("pcm_options", None)
+    if pcm_options:
+        name = f"{molecule}_{basis}_pcm_adc1"
+    else:
+        name = f"{molecule}_{basis}_adc1"
+
+    geom = static_data.xyz[molecule]
+
+    ret = {"basis": basis,
+           "method": "adc1",
+           "molecule": molecule}
+
+    wfn, res = run_psi4_tdscf(geom, basis, pcm_options=pcm_options)
+
+    ret["energy_scf"] = wfn.energy()
+    # yaml safe_dump doesn't like np.floats
+    ret["excitation_energy"] = [round(float(r["EXCITATION ENERGY"]), 5)
```

Comment:
Convert to list:
```Python
list([round(float(r["EXCITATION ENERGY"]), 5) for r in res])
```

Reply:
it is already a list

Reply:
Ah yes, I was confused because of the yaml output stuff... I'm wondering why the print-out looks weird.